multiapps-controller-web/src/main/java/org/cloudfoundry/multiapps/controller/web/configuration/service/ObjectStoreServiceInfoCreator.java

```diff
@@ -46,10 +50,20 @@ private ObjectStoreServiceInfo createServiceInfoForAzure(Map<String, Object> cre
                                               .provider("azureblob")
                                               .identity(accountName)
                                               .credential(sasToken)
+                                              .endpoint(getContainerUriEndpoint(credentials).toString())
                                               .container(containerName)
                                               .build();
     }
 
+    private URL getContainerUriEndpoint(Map<String, Object> credentials) {
+        try {
+            URL containerUri = new URL((String) credentials.get("container_uri"));
+            return new URL(containerUri.getProtocol(), containerUri.getHost(), containerUri.getPort(), "/");
```

Comment:
Does the `getHost()` method return the domain as well?
And why do we need a forwards slash at the end if the path isn't provided?

Reply:
yes, it returns everything up to the first slash.
The path will be provided by the jclouds library, it is the container_name